global_shortcut.go

```diff
@@ -0,0 +1,123 @@
+package astilectron
+
+import "context"
+
+const (
+	EventNameGlobalShortcutCmdRegister          = "global.shortcuts.event.registered"
+	EventNameGlobalShortcutCmdIsRegistered      = "global.shortcuts.event.is.registered"
+	EventNameGlobalShortcutCmdUnregister        = "global.shortcuts.event.unregistered"
+	EventNameGlobalShortcutCmdUnregisterAll     = "global.shortcuts.event.unregistered.all"
+	EventNameGlobalShortcutEventProcessFinished = "global.shortcuts.event.process.finished" // Register or Unregister process is finished
+	EventNameGlobalShortcutEventTriggered       = "global.shortcuts.event.triggered"
+)
+
+type callback func()
+
+// GlobalShortcuts represents a global shortcut
+type GlobalShortcuts struct {
+	*object
+	callbacks map[string]*callback // Store all registered Global Shortcuts
+}
+
+func newGlobalShortcuts(ctx context.Context, d *dispatcher, i *identifier, w *writer) (gs *GlobalShortcuts) {
+	var obj = newObject(ctx, d, i, w, i.new())
+	gs = &GlobalShortcuts{object: obj, callbacks: make(map[string]*callback)}
```

Comment:
Could you avoid creating the `var obj` and instead `gs = &GlobalShortcuts{object: newObject(ctx, d, i, w, i.new()), ...}` and below `gs.On(...)`?